res/css/structures/_UserMenu.pcss

```diff
@@ -25,8 +25,6 @@ limitations under the License.
 
         .mx_UserMenu_userAvatar {
             position: relative;
-            /* without this a default avatar will cause this to be 4px oversized and out of alignment */
```

Comment:
Actually: with~out~ this a default avatar will cause this to be 4px oversized and out of alignment :upside_down_face: